src/VisualBasic/CodeCracker/Usage/UnusedParametersCodeFixProvider.vb

```diff
@@ -71,9 +71,15 @@ Namespace Usage
                     Dim arguments = If(objectCreation IsNot Nothing,
                         objectCreation.ArgumentList,
                         methodIdentifier.FirstAncestorOfType(Of InvocationExpressionSyntax).ArgumentList)
-                    If parameter.Modifiers.Any(Function(m) m.IsKind(SyntaxKind.ParamArrayKeyword)) Then
+
+                    ' Attempt to find the parameter as a named argument.  Named arguments can only appear once in the argument list.
+                    Dim namedArg = arguments.Arguments.Where(Function(arg) arg.IsNamed).OfType(Of SimpleArgumentSyntax).SingleOrDefault(Function(arg) arg.NameColonEquals.Name.Identifier.Text = parameter.Identifier.Identifier.Text)
```

Comment:
I'm not sure if this is the most efficient way to do this.  Other ideas are appreciated.